package.json

```diff
@@ -65,12 +66,11 @@
     "react": "16.11.0",
     "react-native": "0.62.2",
     "react-native-macos": "0.60.0-microsoft.73",
-    "react-native-windows": "^0.62.5",
-    "semantic-release": "15.13.24",
-    "typescript": "3.8.3",
-    "appium": "1.17.0",
+    "react-native-windows": "^0.63.11",
     "selenium-appium": "0.0.15",
-    "selenium-webdriver": "4.0.0-alpha.7"
+    "selenium-webdriver": "4.0.0-alpha.7",
+    "semantic-release": "15.13.24",
+    "typescript": "3.8.3"
```

Comment:
appium, selenium, semantic-release, and typescript were just moved, not changed -- automatically I guess? Alphabetically now.

Only thing that changes is react-native-windows version from 0.62.5 to 0.63.11.